crates/ra_cargo_watch/src/lib.rs

```diff
@@ -316,6 +316,8 @@ impl WatchThread {
             args.push("--all-targets".to_string());
         }
         args.extend(options.args.iter().cloned());
+        args.sort();
```

Comment:
This will mess up `--package foo`, we should not sort flags. In general, a good rule of thumb is to use input things as is. For example, `cargo` deliberately avoids canonicalizing and/or normalizing paths received from user.